cmd/frontend/internal/app/templates/ui/app.html

```diff
@@ -31,6 +40,12 @@
 </head>
 
 <body>
+	{{ if .Context.SourcegraphDotComMode }}
+	<!-- Google Tag Manager (noscript) -->
+	<noscript><iframe ignore-csp src="https://www.googletagmanager.com/ns.html?id=GTM-TB4NLS7"
+	height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
```

Comment:
I think the problem is that it is not on the same line (the `ignore-csp` and the `style`)
```suggestion
	<noscript><iframe ignore-csp src="https://www.googletagmanager.com/ns.html?id=GTM-TB4NLS7" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
```

Reply:
Thanks @felixfbecker